tools/proto-js-plugin/src/test/java/io/spine/js/generate/imports/ResolveImportsTest.java

```diff
@@ -148,4 +150,10 @@ private static IterableSubject afterResolve(Path file, ResolveImports task) thro
     private ResolveImports newTask(ExternalModule module) {
         return new ResolveImports(generatedProtoDir, ImmutableList.of(module));
     }
+
+    private static ExternalModule newModule(String moduleName, String directoryPattern) {
```

Comment:
It's weird that this file has just this one `private static` method appended. Please review.